data/systemd/anaconda-nm-disable-autocons-rhel.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+ConditionKernelCommandLine=|ip
+ConditionKernelCommandLine=|inst.ks
+ConditionOSRelease=ID=rhel
```

Comment:
This works for RHEL, unfortunately not for CentOS Stream, see https://issues.redhat.com/browse/RHEL-67815